src/components/PostsList/PostsList.scss

```diff
@@ -0,0 +1,34 @@
+.PostsList {
+  width: 100%;
+
+  &__user-button {
+    cursor: pointer;
+    padding: 5px 10px;
+    border-radius: 5px;
+    font-weight: bold;
+
+    &:hover {
+      background-color: lightblue;
+    }
+  }
+
+  &__list {
+    list-style: none;
+    padding-left: 5px;
+  }
+
+  &__item {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-radius: 10px;
+    padding: 10px;
+    margin-bottom: 5px;
+    background-color: #fafafd;
```

Comment:
Suggestion: insert colors in a `variables.scss` or `colors.scss` file, and import it in the `App.scss` file.